pregenerate unicode files; mktables uses too much memory for some platforms
OpenBSD Revision 1.21 of Makefile.SH

Need to run mktables manually and then update MANIFEST and apply this patch

diff -Nrux *.orig ../perl-5.16.0.orig/Makefile.SH ./Makefile.SH
--- ../perl-5.16.0.orig/Makefile.SH	Mon May 14 12:49:21 2012
+++ ./Makefile.SH	Mon Jun  4 20:31:43 2012
@@ -1039,10 +1039,8 @@
 unidatafiles $(unidatafiles) pod/perluniprops.pod: uni.data
 
 uni.data: $(MINIPERL_EXE) $(CONFIGPM) lib/unicore/mktables $(nonxs_ext)
-	$(MINIPERL) $(Icwd) lib/unicore/mktables -C lib/unicore -P pod -maketest -makelist -p
-#	Commented out so always runs, mktables looks at far more files than we
-#	can in this makefile to decide if needs to run or not
-#	touch uni.data
+#	$(MINIPERL) $(Icwd) lib/unicore/mktables -C lib/unicore -P pod -maketest -makelist -p
+	touch uni.data
 
 # $(PERL_EXE) and ext because buildtoc uses Text::Wrap uses re
 # But also this ensures that all extensions are built before we try to scan
